src/main/java/com/google/jenkins/plugins/k8sengine/KubeConfig.java

```diff
@@ -114,23 +120,25 @@ public Builder currentContext(String currentContext) {
       return this;
     }
 
-    public Builder users(ImmutableList<Object> users) {
+    public Builder users(List<Object> users) {
```

Comment:
this does not appear to be binary compatible and is the public interface.  Have you checked for external callers of this function?

Reply:
in in this case it's binary compatible as ImmutableList is an implementation of List